packages/sdk/src/conversation/interface.ts

```diff
@@ -173,6 +173,78 @@ export interface CommandOptions {
   commands?: TeamsFxBotCommandHandler[];
 }
 
+/**
+ * Options to initialize {@link CardActionBot}.
+ */
+export interface CardActionOptions {
+  /**
+   * The action handlers to registered with the action bot. Each command should implement the interface {@link TeamsFxAdaptiveCardActionHandler} so that it can be correctly handled by this bot.
+   */
+  actions?: TeamsFxAdaptiveCardActionHandler[];
+}
+
+/**
+ * Options used to control how the response card will be sent to users.
+ */
+export enum AdaptiveCardResponse {
+  /**
+   * The response card will be replaced the current one for the interactor who trigger the action.
+   */
+  ReplaceForInteractor,
+
+  /**
+   * The response card will be replaced the current one for all users in the chat.
+   */
+  ReplaceForAll,
+
+  /**
+   * The response card will be sent as a new message for all users in the chat.
+   */
+  NewForAll,
+}
+
+/**
+ * Interface for adaptive card action handler that can process card action invoke and return a response.
+ */
+export interface TeamsFxAdaptiveCardActionHandler {
+  /**
+   * The verb defined in adaptive card action that can trigger this handler.
+   */
+  triggerVerb: string;
```

Comment:
Is the verb case sensitive/insensitive? Better to add to doc.

Reply:
We'd better make it case-insensitive. Let me update it.